cpp/tests/join/join_tests.cu

```diff
@@ -580,9 +580,9 @@ typedef ::testing::Types<
                           TestParameters< join_op::FULL, HASH, VTuple<int32_t > >,
                           TestParameters< join_op::FULL, HASH, VTuple<int64_t > >,
                           TestParameters< join_op::FULL, HASH, VTuple<float   > >,
-                          TestParameters< join_op::FULL, HASH, VTuple<double  > >,
```

Comment:
Are you removing hash-based FULL join tests for these three datatypes? Shouldn't you just be adding tests for sort-based join, not removing the hash-based tests?

Reply:
Unfortunately, there is a limit to the number of template arguments in testing::Types. A few lines were removed to avoid compilation errors.

Reply:
Sounds reasonable. I don't want to leave functionality untested.